src/main/scala/Exercise8_1.scala

```diff
@@ -0,0 +1,28 @@
+object Exercise8_1 extends App {
+  class BankAccount(initialBalance: Double) { // provided in the exercise 8.1
+    private var balance = initialBalance // I cannot access or override it in subclasses!
+    def currentBalance = balance
+    def deposit(amount: Double) = {
+      balance += amount;
```

Comment:
you don't need `;` at the end of the line

Reply:
IDEA writes it automatically and I forgot to erase it